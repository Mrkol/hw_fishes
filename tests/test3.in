1
......
.....C
.FFF.C
.....C
......
