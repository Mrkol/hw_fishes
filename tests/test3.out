......
..F...
C.F.CC
..F...
......
